### PR TITLE
[cordova][xwalk-4226] Update the app function cases for gallery

### PR DESCRIPTION
**Pull request by zhuyongyong**
- Add the gallery sample's test step and expect result for 2 impact cases
- Consist the component names for 'Cordova Features' test suites

Impacted tests(approved): new 0, update 2, delete 0
Unit test platform: [Android]
Unit test result summary: pass 2, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-4226
BUG=https://crosswalk-project.org/jira/browse/XWALK-3996

### CODE REVIEW

File: cordova/cordova-sampleapp-android-tests/sampleapp/Crosswalk_Cordova_SampleApp_Gallery_AppFunction.html [hunk 1]
```diff
@@ -41,12 +42,19 @@
       <strong>Test steps:</strong>
     </p>
     <ol>
+      <li>Check the display of "Carousel image gallery" from Gallery UI</li>
```
Comment (Honry) [inline cordova/cordova-sampleapp-android-tests/sampleapp/Crosswalk_Cordova_SampleApp_Gallery_AppFunction.html]: There should be a precondition to access a good network.

File: misc/sampleapp-android-tests/sampleapp/gallery_appFunction.html [hunk 2]
```diff
@@ -41,13 +42,19 @@
       <strong>Test steps:</strong>
     </p>
     <ol>
-      <li>Check every button from Gallery UI.</li>
```
Comment (Honry) [inline misc/sampleapp-android-tests/sampleapp/gallery_appFunction.html]: Ditto

**Comment by Honry** [comment]
Please provide the UT result.

**Comment by Honry** [comment]
@cicili, we also update the Gallery tests in sampleapp-android-tests, please help to review.

**Comment by cicili** [comment]
LGTM

**Pull request merged**